### ISSUE TITLE
Chromaticraft: Focus Crystal progression required but not labled in lexicon nor changelog

### ISSUE THREAD

**Issue by thePalindrome** [ReikaKalseki/Reika_Mods_Issues#2462]
v19a (Nov 9 2017):
> ChromatiCraft: Focus Crystal progression is now optional

Sometime in 2019, this was changed, but the lexicon makes no indication that progression is required, and the changelog does not mention any changes to the focus crystal progression.

EDIT: Reika has claimed there is an entry for v22a stating:
> ChromatiCraft: Changed progression related to repeater development

However, this is not only vague, it does not appear at all in the public changelog

**Comment by ReikaKalseki** [comment]
The focus crystals are not *explicitly* needed for another progression; they merely are a required fragment at a given point in the research tiers, thus meaning that they are an indirect prerequisite of everything on later tiers. This is not unique, and there is currently no way of showing such indirect relationships, nor any good ideas on hand.

**Comment by thePalindrome** [comment]
I'd argue that then implies that the progression system itself needs a reworking, since it's become a bit of a gordion knot at this point :P 
As it is, just something to keep in mind. If the progression system as it is can cause huge confusion like this, it *may* be worthwhile to revamp it

**Comment by tiny-maple** [comment]
![2020-04-29_01 46 23](https://user-images.githubusercontent.com/62176592/80668977-cff79d80-8ad5-11ea-9a86-f77f54028ff4.png)
![2020-04-29_02 13 44](https://user-images.githubusercontent.com/62176592/80669363-d2a6c280-8ad6-11ea-82c5-088f58ed0fdc.png)

Since the focus crystal progression was optional, i only went to obtain them after I restored the monument, as i justify a lack of incentive to trade with villagers. Also I did not really notice the the improvement to the casting speed, since every cast I would go ahead and do something else, etc setup wireless redstone control.

If there was more incentive to trade with villagers for the focus crystal. for example turbochraged exquisite focus crystal would speed up casting so fast that casting for a stack of Energy focal point would instant cast it, I would invest my time to obtain these crystals then upgrade all my structures with focus crystals.
